src/families/bitcoin/wallet-btc/crypto/decred.ts

```diff
@@ -7,8 +7,7 @@ import createBlakeHash from "blake-hash";
 import RIPEMD160 from "ripemd160";
 import bs58 from "bs58";
 import ecc from "tiny-secp256k1";
-import createHash from "create-hash";
-import createHmac from "create-hmac";
+import crypto from "crypto";
```

Comment:
Can you please check this works on LLM? Since `crypto` is a node.js package I'm thinking it may not work in react-native.

Reply:
I've checked with LLM and LLD. The PR is ready to be merged